lib/tasks_private/spec_hawkular.rake

```diff
@@ -0,0 +1,136 @@
+require 'net/http'
+require 'json'
+require 'hawkularclient'
+
+module ManageIQ::Providers::Hawkular::TaskHelpers
+  class LiveSetup
```

Comment:
Being honest, I also had my doubts about how to organize the code. At the end, I took the approach I saw at [evm_settings.rake](https://github.com/ManageIQ/manageiq/blob/master/lib/tasks/evm_settings.rake) where they have the module in the rake file. It make sense because that module is relevant only for the accompanying rake task.

But I think having a `task_helpers` directory seems reasonable. Let me know if you are better with that approach so that create the directory and I move the module there.